### ISSUE TITLE
ArgumentsAttribute causes an error when used with a string containing quotes

### ISSUE THREAD

**Issue by UoLeevi** [dotnet/BenchmarkDotNet#821]
Following causes an error when benchmark is run.<br/>
`
[Benchmark]
[Arguments("\"")]
public void BugBenchmark(string arg) {  }
`

**Comment by Rizzen** [comment]
Could you provide description of error? I have no repro.

**Comment by UoLeevi** [comment]
`// ***** BenchmarkRunner: Start   *****
// Found benchmarks:
//   StringArgBugBenchmarks.BugBenchmark: Core(Runtime=Core) [arg="]

// Validating benchmarks:
// ***** Building 1 benchmark(s) in Parallel: Start   *****
// ***** Done, took 00:00:04 (4.33 sec)   *****
// Build Exception: Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  MailPress -> C:\dev\MailPress\MailPress\bin\Release\netcoreapp2.1\MailPress.dll
  MailPress.Benchmarks -> C:\dev\MailPress\MailPress.Benchmarks\bin\Release\netcoreapp2.1\MailPress.Benchmarks.dll
460b52bd-e400-4a0e-a37b-7731f5efcdae.notcs(151,29): error CS1002: ; expected [C:\dev\MailPress\MailPress.Benchmarks\bin\Release\netcoreapp2.1\460b52bd-e400-4a0e-a37b-7731f5efcdae\BenchmarkDotNet.Autogenerated.csproj]
460b52bd-e400-4a0e-a37b-7731f5efcdae.notcs(151,29): error CS1010: Newline in constant [C:\dev\MailPress\MailPress.Benchmarks\bin\Release\netcoreapp2.1\460b52bd-e400-4a0e-a37b-7731f5efcdae\BenchmarkDotNet.Autogenerated.csproj]
460b52bd-e400-4a0e-a37b-7731f5efcdae.notcs(151,31): error CS1002: ; expected [C:\dev\MailPress\MailPress.Benchmarks\bin\Release\netcoreapp2.1\460b52bd-e400-4a0e-a37b-7731f5efcdae\BenchmarkDotNet.Autogenerated.csproj]

Build FAILED.

460b52bd-e400-4a0e-a37b-7731f5efcdae.notcs(151,29): error CS1002: ; expected [C:\dev\MailPress\MailPress.Benchmarks\bin\Release\netcoreapp2.1\460b52bd-e400-4a0e-a37b-7731f5efcdae\BenchmarkDotNet.Autogenerated.csproj]
460b52bd-e400-4a0e-a37b-7731f5efcdae.notcs(151,29): error CS1010: Newline in constant [C:\dev\MailPress\MailPress.Benchmarks\bin\Release\netcoreapp2.1\460b52bd-e400-4a0e-a37b-7731f5efcdae\BenchmarkDotNet.Autogenerated.csproj]
460b52bd-e400-4a0e-a37b-7731f5efcdae.notcs(151,31): error CS1002: ; expected [C:\dev\MailPress\MailPress.Benchmarks\bin\Release\netcoreapp2.1\460b52bd-e400-4a0e-a37b-7731f5efcdae\BenchmarkDotNet.Autogenerated.csproj]
    0 Warning(s)
    3 Error(s)

Time Elapsed 00:00:00.90`

**Comment by Rizzen** [comment]
Yep, I have repro on 0.10.14, but it seems to be fixed in 0.11.0.
You can try package from CI source - that described here - https://github.com/dotnet/BenchmarkDotNet/issues/817#issuecomment-402186579

**Comment by Rizzen** [comment]
It seems also broken in CI source package. But when built from sources it works.
@adamsitnik how old is package in CI source?
